content/docs/searching.md

```diff
@@ -101,29 +101,30 @@ The various action buttons available include:
 
         _The **Previous Selection** column indicates that a range of text has been selected already. The **Wrap Around** and **Backward Direction** and **In Selection** columns refer to the setting of the checkboxes described above. The **Range** column describes the range of the document that is affected by the **Replace All**. A value of "-/-" means that the setting does not influence the outcome for that combination of conditions._
 
-* **Replace All in All Opened Documents**: same as **Replace All**, but goes through all the documents open in Notepad++, not just the active document.
+* **Replace All in All Opened Documents**: Same as **Replace All**, but goes through all the documents open in Notepad++, not just the active document.
 
-The above actions may be initiated via mouse by pressing the appropriate button, or via special `Alt` key combinations.  With focus on one of the Find dialog windows, press and release the `Alt` key.  This will cause Notepad++ to underline a single character in the text of *most* of the buttons.  Pressing Alt and one of the underlined characters will be the same as pressing the same button with the mouse, i.e., the chosen action will be initiated.  The Alt technique works for controls other than buttons as well, e.g., a checkbox control can be checked/unchecked via pressing its Alt key command.
+The above actions may be initiated via mouse by pressing the appropriate button, or via special `Alt` key combinations.  Notepad++ underlines a single character in the text of *most* of the buttons.  Pressing `Alt` and one of the underlined characters will be the same as pressing the same button with the mouse, i.e., the chosen action will be initiated.  The `Alt` technique works for controls other than buttons as well, e.g., a checkbox control can be checked/unchecked via pressing its `Alt` key command.  In a similar vein, pressing `Alt` outside one of the Find dialog windows underlines items in the top main menu.
```

Comment:
Your "correction" isn't entirely accurate.  But neither is the original phraseology.

There are times when bringing up one of the search menus does **not** show any of the underlines on one of the buttons: when that happens, clicking `Alt` alone _will_ show the missing underlines.  But the missing-underline-condition doesn't happen every time, which is what the original phrasing implied.

A better phrasing would be:
> Notepad++ can underline a single character in the text of *most* of the buttons (oftentimes, it is automatic; however, if you ever cannot see the underlines, then just click the `Alt` key, and they will appear).

You also added:
> In a similar vein, pressing `Alt` outside one of the Find dialog windows underlines items in the top main menu.

That is _not_ the point of the original comment about press-and-release the `Alt` key, and does not belong in this page of the manual.  (And since it's common Windows behavior, doesn't really need to be in the user manual at all.)

Reply:
I see. I wonder how at times it works and then it doesn't. It seems to work consistently on my end.
Copied your improvement. Deleted the last part which I thought was what was meant.